xobjects/hybrid_class.py

```diff
@@ -373,3 +381,43 @@ def __repr__(self):
 
 class ThisClass:  # Place holder
     pass
+
+
+class PyMethod:
+
+    def __init__(self, kernel_name, element, additional_arg_names):
+        self.kernel_name = kernel_name
+        self.element = element
+        self.additional_arg_names = additional_arg_names
+
+    def __call__(self, **kwargs):
+        instance = self.element
+        context = instance.context
```

Comment:
Still waiting for all the tests, but it seems that this should have been `instance._context`.